Assets/Scripts/EncounterScripts/EncounterManager.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public enum EncounterOutcome
+{
+	Victory,
+	Defeat
+}
+
+public class EncounterManager : MonoBehaviour
+{
+	private Encounter activeEncounter; // Currently activated Encounter
+
+	/// <summary>
+	/// Enter Combat
+	/// </summary>
+	/// <param name="encounter"></param>
+	public void TriggerEncounter(Encounter encounter)
+	{
+		if (activeEncounter != null)
+		{
+			Debug.LogWarning("An encounter is already active!");
+			return;
+		}
+		if (encounter)
+		{
+			activeEncounter = encounter;
+			activeEncounter.StartEncounter();
+		}
+		else
+		{
+			Debug.LogWarning("There is no ENCOUNTER DATA");
+		}
+	}
+
+	/// <summary>
+	/// Finish Combat
+	/// </summary>
+	/// <param name="outcome"></param>
+	public void OnEncounterComplete(EncounterOutcome outcome)
```

Comment:
is encounter outcome as an enum necessary? i feel like "bool victory" would accomplish the same thing as we're never gonna have a third outcome